### ISSUE TITLE
Can't register custom cell

### ISSUE THREAD

**Issue by abraham-vineti** [eclipsesource/jsonforms#1527]
**Describe the bug**
 After creating a simple custom cell and registering it, it never seems to get used.

I created a [`SampleCell`](https://github.com/abraham-vineti/jsonforms-react-seed/pull/1) based on [`MaterialBooleanCell`](https://github.com/eclipsesource/jsonforms/blob/master/packages/material/src/cells/MaterialBooleanCell.tsx) in the [`jsonforms-react-seed`](https://github.com/eclipsesource/jsonforms-react-seed) app.

**To Reproduce**
1. Check out the `cell` branch from this [`jsonforms-react-seed` fork](https://github.com/abraham-vineti/jsonforms-react-seed/pull/1)
2. npm install
3. npm start
4. Expect custom cell to be rendered

**Expected behavior**
I expect `SAMPLE` from the cell to be rendered or `SAMPLE` from the `console.log` to be in the browser dev tools.

**Browser (please complete the following information):**
 - Chrome 78
 - Firefox 70

**Used Setup (please complete the following information):**
 - [jsonforms-react-seed](https://github.com/eclipsesource/jsonforms-react-seed)

**Additional context**
I'm working on upgrading `jsonforms` for a large project that has a custom field. After updating the field to be a cell it won't render.

**Comment by sdirix** [comment]
Thanks for creating this issue and adding so much information. We'll have a look at the problem soon.

**Comment by sdirix** [comment]
Hi @abraham-vineti, I quickly checked the provided example. The cell is not rendered because cells are only used in the array renderers. So when you for example use the following schema and uischema your cell renderer will be triggered:

```json
{
  "type": "object",
  "properties": {
    "name": {
      "type": "string",
      "minLength": 1
    },
    "description": {
      "type": "string"
    },
    "done": {
      "type": "boolean"
    },
    "due_date": {
      "type": "string",
      "format": "date"
    },
    "rating": {
      "type": "integer",
      "maximum": 5
    },
    "recurrence": {
      "type": "string",
      "enum": ["Never", "Daily", "Weekly", "Monthly"]
    },
    "recurrence_interval": {
      "type": "integer"
    },
    "myarray": {
      "type": "array",
      "items": {
        "type": "object",
        "properties": {
          "firstProp": {
            "type": "string"
          },
          "secondProp": {
            "type": "integer"
          }
        }
      }
    }
  },
  "required": ["name", "due_date"]
}
```
```json
{
  "type": "VerticalLayout",
  "elements": [
    {
      "type": "Control",
      "label": false,
      "scope": "#/properties/done"
    },
    {
      "type": "Control",
      "scope": "#/properties/name"
    },
    {
      "type": "HorizontalLayout",
      "elements": [
        {
          "type": "Control",
          "scope": "#/properties/due_date"
        },
        {
          "type": "Control",
          "scope": "#/properties/rating"
        }
      ]
    },
    {
      "type": "Control",
      "scope": "#/properties/description",
      "options": {
        "multi": true
      }
    },
    {
      "type": "HorizontalLayout",
      "elements": [
        {
          "type": "Control",
          "scope": "#/properties/recurrence"
        },
        {
          "type": "Control",
          "scope": "#/properties/recurrence_interval",
          "rule": {
            "effect": "HIDE",
            "condition": {
              "type": "LEAF",
              "scope": "#/properties/recurrence",
              "expectedValue": "Never"
            }
          }
        }
      ]
    },
    {
      "type": "Control",
      "scope": "#/properties/myarray"
    }
  ]
}
```
![myarray](https://user-images.githubusercontent.com/8998368/70058007-6cc51600-15de-11ea-9ac4-338210530cb5.png)

**Comment by abraham-vineti** [comment]
Is that a change from how fields worked `v2.2.0`? I've got a codebase that seems to be rendering `Fields` for everything. They only stopped working after upgrading and renaming to `Cells`.

**Comment by sdirix** [comment]
Yes we replaced `fields` with `cells` and changed how they work in this PR https://github.com/eclipsesource/jsonforms/pull/1320 released with `v2.2.3`. Now cells and normal controls are handled separately.

**Comment by sdirix** [comment]
I will close this for now. Let me know when you have further questions.
